crates/evm-prover/src/grpc/server.rs

```diff
@@ -17,6 +21,21 @@ pub async fn create_grpc_server(config: Config) -> Result<()> {
         .build()
         .unwrap();
 
+    // TODO: Remove this config cloning when we can rely on the public key from config
+    // https://github.com/evstack/ev-node/issues/2603
+    let mut config_clone = config.clone();
+    config_clone.pub_key = public_key().await?;
+    println!("Successfully got pubkey from evnode: {}", config_clone.pub_key);
+
+    tokio::spawn({
```

Comment:
I think we should rename this crate from grpc -> server and add shutdown cancellation/signal handling properly to these long lived tasks. Can do this in a follow up

Reply:
https://github.com/celestiaorg/celestia-zkevm-hl-testnet/issues/152